src/commands/general/caniuse.ts

```diff
@@ -0,0 +1,44 @@
+import CommandInt from "@Interfaces/CommandInt";
+import { MessageEmbed } from "discord.js";
+
+const caniuse: CommandInt = {
+  name: "caniuse",
+  description:
+    "Returns an up-to-date browser support table for a feature",
+  parameters: ["`<feature>`: the browser feature to define"],
+  run: async (message) => {
+    try {
+      const { channel, commandArguments } = message;
+
+      const feature = commandArguments.shift();
```

Comment:
```suggestion
      const feature = commandArguments.join("-");
```
`.shift()` will only take the first argument - in this case, we want all of them.